nemo/core/classes/common.py

```diff
@@ -356,6 +356,7 @@ def from_pretrained(
         model_name: str,
         refresh_cache: bool = False,
         override_config_path: Optional[str] = None,
+        override_data_dir: Optional[str] = None,
```

Comment:
why this parameter was changed. Can't we control this via setup_training/validation_data methods?

Reply:
Also, data_dir is in config file which can be overridden right now anyways.

Reply:
yes, just mixed with other use case